apps/chart-docsite/package.json

```diff
@@ -21,13 +21,10 @@
   },
   "dependencies": {
     "@fluentui/react-charts-preview": "*",
-    "@fluentui/react-components": "*",
-    "@griffel/react": "^1.5.22",
-    "@microsoft/applicationinsights-web": "^3",
+    "@fluentui/react-storybook-addon": "*",
```

Comment:
>fluentui [](http://example.com/codeflow?start=6&length=8)

Thanks for making this change.
There is 1 issue. The PR workflow is not getting triggered for master CI.
[https://fluentuipr.z22.web.core.windows.net/heads/master/](https://fluentuipr.z22.web.core.windows.net/heads/master/) are pointing to old deployments and charts-docsite is not showing up there.

Reply:
not sure I follow ? everything is deployed as expected on PR run 

https://fluentuipr.z22.web.core.windows.net/pull/33811/public-docsite-v9/storybook/index.html?path=/docs/charts_charts-legends--docs

note: there have been some cache issues within GHA yesterday